src/styles/blocks/card.scss

```diff
@@ -0,0 +1,159 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  place-content: center;
+  place-items: center;
+}
+
+@media (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+  margin: 0; /* Remove any default margins */
+  transition: transform 300ms ease; /* Simplified transition */
+
+  /* Rest of the .product-card styles remain the same */
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
```

Comment:
The values for `margin` should follow the single space format after the colon as per the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). It should be `margin: 32px  19px  40px;`.